### ISSUE TITLE
Arch Linux Support

### ISSUE THREAD

**Issue by NullEntity** [XereoNet/SpaceBukkitPanel#11]
I've had a bit of difficulty with getting Spacebukkit to work under Arch, but I managed to do so by disabling caching and setting the error level to E_ALL ^ E_STRICT. Without it I got an error stating: Fatal error: Class 'String' not found in /srv/http/admin/app/Model/User.php on line 40 when trying to access Spacebukkit after installation. I believe File caching could still be used by creating the "persistent" and "models" folders with proper permissions in the install.sh file. I believe the line blocking installation in /srv/ should be removed. Arch installs Apache's http dir to /srv/http. Another option would be to add an option asking if you would like to continue anyway.

**Comment by gdude2002** [comment]
Yeah, IMO the install script is a little more idiot-proof than is necessary. Nginx on our Arch is set up in /etc/nginx :P
